Cargo.toml

```
@@ -15,6 +15,8 @@ built = { version = "0.7.7", features = ["git2"] }
 chrono = "0.4.39"
 clap = { version = "4.5.29", features = ["derive"] }
 colored = "2.2.0"
+codspeed = { package = "codspeed-criterion-compat", version = "2.8.0" }
+criterion = "0.5.1"
```

```suggestion
criterion = { package = "codspeed-criterion-compat", version = "2.8.0" }
```

これはFeature導入せずにこのままで、ローカルで実行する時も `codspeed-criterion-compat` は `criterion` 本体と同様に動作すると思いますが、何か挙動が違う部分がありましたか？

---

cargo-codspeed だと手元で動かなかった（ARM Mac をサポートしていないため）ので使えないものだと早合点していましたが、普通に cargo bench を呼べば使えるんですね。feature 廃止しようと思います :+1: